examples/Advanced/Advanced.ino

```diff
@@ -0,0 +1,47 @@
+/*
+    WifiSettings advanced example
+
+    Demonstrates callback functions and custom variables
+    to be saved through WifiSettings.
+
+    Source and further documentation available at
+    https://github.com/Juerd/esp32-WiFiSettings
+*/
+
+#include <SPIFFS.h>
+#include <WiFi.h>
+#include <WiFiSettings.h>
+
+void setup() {
+    Serial.begin(115200);
+    SPIFFS.begin(true);  // Will format on the first run after failing to mount
+
+    // Set custom callback functions
+    WiFiSettings.onSuccess  = []() {
+        Serial.println("Successfully connected to WiFi");
+    };
+    WiFiSettings.onFailure  = []() {
+        Serial.println("Failed to connect to WiFi!");
+    };
+    WiFiSettings.onWaitLoop = []() {
+        Serial.print(".");
+        return 500; // Delay next function call by 1000ms
+    };
```

Comment:
The library itself already prints stuff to serial, including the dots.

Reply:
Fair. I've updated the advanced example to use an LED.